internal/pkg/cli/pipeline_init.go

```diff
@@ -247,14 +247,36 @@ func (o *initPipelineOpts) createPipelineProvider() (manifest.Provider, error) {
 	return manifest.NewProvider(config)
 }
 
+func (o *initPipelineOpts) getEnvFromCache(environmentName string) (*config.Environment, error) {
+	var environment *config.Environment
+	for _, env := range o.envs {
+		if env.Name == environmentName {
+			environment = env
```

Comment:
nit: `break` after this statement so that we don't continue looping.

Reply:
I addressed.

Reply:
nit: oh my bad, instead of breaking we should just return instead :)

Reply:
and then outside of the for-loop we wouldn't need to check if it's equal to nil and return also an error immediately

Reply:
Totally agree! I addressed.